store/details_v2_test.go

```diff
@@ -324,9 +323,11 @@ func (s *detailsV2Suite) TestInfoFromStoreSnap(c *C) {
 				checker(pfx+f.Name+".", v)
 				continue
 			}
+			name := pfx + f.Name
 			if reflect.DeepEqual(v.Interface(), reflect.Zero(f.Type).Interface()) {
-				name := pfx + f.Name
 				c.Check(expectedZeroFields, testutil.Contains, name, Commentf("%s not set", name))
+			} else {
```

Comment:
thanks for adding this additional check, hopefully it should make it more obvious in the future